### PR TITLE
Adding support for nestedDepth option to getResources method.

### LINKED ISSUE #410: List nested resources in a stack
Using the `client.getResources` method ([documentation](https://github.com/pkgcloud/pkgcloud/blob/master/docs/providers/rackspace/orchestration.md#clientgetresourcesstack-callback) | [implementation](https://github.com/pkgcloud/pkgcloud/blob/master/lib/pkgcloud/openstack/orchestration/client/resources.js#L56-L96)) a user can list resources in a stack. This method, however, does not allow the user to list *nested* resources in the stack.

In the underlying HTTP API (`GET /stacks/{id}/resources`), nested resources can be requested by supplying the `nested_depth` query parameter ([documentation](http://developer.openstack.org/api-ref-orchestration-v1.html#stack-resources)).

### PR DESCRIPTION
**Pull request by ycombinator**
Fixes #410.

### CODE REVIEW

File: lib/pkgcloud/openstack/orchestration/client/resources.js [hunk 1]
```diff
@@ -62,13 +62,25 @@ exports.getResource = function (stack, resource, callback) {
...
+  }
+
   function getResources(stack) {
-    return self._request({
+    var requestOptions = {
       path: urlJoin('/stacks', stack.name, stack.id, 'resources')
-    }, function (err, body) {
+    };
+
+    requestOptions.qs = [];
```
Comment (kenperkins) [inline lib/pkgcloud/openstack/orchestration/client/resources.js]: Why not move this initialization into the definition of `requestOptions` on line 74?

Comment (ycombinator) [inline lib/pkgcloud/openstack/orchestration/client/resources.js]: Yeah, duh. Changing.

File: lib/pkgcloud/openstack/orchestration/client/resources.js [hunk 2]
```diff
@@ -62,13 +62,25 @@ exports.getResource = function (stack, resource, callback) {
...
+    if (options.nestedDepth) {
+      requestOptions.qs['nested_depth'] = options.nestedDepth;
```
Comment (kenperkins) [inline lib/pkgcloud/openstack/orchestration/client/resources.js]: Should we do a `typeof` check here? We're not very consistent on that.

Comment (ycombinator) [inline lib/pkgcloud/openstack/orchestration/client/resources.js]: You mean for whether `options.nestedDepth` is an integer?

Comment (kenperkins) [inline lib/pkgcloud/openstack/orchestration/client/resources.js]: yea. I think it's probably fine.

**Comment by ycombinator** [comment]
@kenperkins I've made the change you requested. Anything else or can this PR be merged?

**Comment by kenperkins** [comment]
lgtm :+1:

**Pull request merged**